### PR TITLE
Added tooltip

### PR DESCRIPTION
**Pull request by Caranell**
Fix issue #73 

### **Before**
Screenshot 2022-08-14 at 11 20 06

### **After**
Screenshot 2022-08-14 at 11 17 30
Screenshot 2022-08-14 at 11 06 19
Screenshot 2022-08-14 at 11 06 14

### CODE REVIEW

File: ui/pages/liquidity.tsx [hunk 1]
```diff
@@ -338,7 +338,12 @@ export default function Liquidity() {
                       className="btn btn-primary normal-case font-medium w-full"
                       action={withdrawAndClaimRewards}
                     >
-                      Withdraw all and claim
```
Comment (gregfromstl) [inline ui/pages/liquidity.tsx:341]: We definitely still want the button label

File: ui/pages/liquidity.tsx [hunk 2]
```diff
@@ -338,7 +338,12 @@ export default function Liquidity() {
...
+                      <div
```
Comment (gregfromstl) [inline ui/pages/liquidity.tsx:341]: The tooltip gets cut off
Screen Shot 2022-08-13 at 9 16 48 PM

**Review by gregfromstl — CHANGES_REQUESTED** [review]

**Comment by aahna-ashina** [comment]
@Caranell Could you include before and after screenshots to make it easier/faster for reviewers to see your UI change?

Here is an example of how to do it:  https://github.com/nation3/app/pull/75

File: ui/pages/liquidity.tsx (hunk 1 above)
Comment (Caranell) [reply ui/pages/liquidity.tsx:341]: Oops, my bad. Didnt mean to delete it

File: ui/pages/liquidity.tsx (hunk 2 above)
Comment (Caranell) [reply ui/pages/liquidity.tsx:342]: Fixed tooltip overflow 👍

**Comment by Caranell** [comment]
> @Caranell Could you include before and after screenshots to make it easier/faster for reviewers to see your UI change?
> 
> Here is an example of how to do it: #75

Sure! Added before/after screens

**Comment by gregfromstl** [comment]
@Caranell are you able to run the integration tests locally? It looks like a test is failing, but I don't think its related to your changes

**Comment by Caranell** [comment]
@gregfromstl Yea, it's weird because locally both headless and normal test work properly for me
 ```
 Nation3 App UI
    ✓ should load the index page (964ms)
    ✓ should load the /join page (684ms)
    ✓ should load the /claim page (2580ms)
    ✓ should load the /lock page (1862ms)
    ✓ should load the /liquidity page (240ms)


  5 passing (6s)
```

**Review by aahna-ashina — APPROVED** [review]

**Comment by aahna-ashina** [comment]
@Caranell Suggestion (out of scope for this PR):  Would it be possible to automatically set the `Amount` input field to the account's token balance when the "Withdraw all and claim" button is clicked?

**Comment by Caranell** [comment]
@aahna-ashina I don't quite understand what you mean 🤔 

Why would you want to show the current user's balance in 'unstake' form? It is possible to switch the tab to 'stake', but again -- it's hard to imagine a situation when the user has withdrawn all of his tokens and for some reason wants to stake right away 🤔

**Comment by aahna-ashina** [comment]
@Caranell I meant for showing the actual amount being unstaked _while_ the user is waiting for MetaMask to open.

**Review by gregfromstl — APPROVED** [review]

**Comment by aahna-ashina** [comment]
> P.S. also added missing goerli contract' addresses

@Caranell Thank you.  Note that those addresses were also added in this PR:  https://github.com/nation3/app/pull/80/files#diff-a60637e7c79e64c6d71a0d3d6c6a6afeee55694d775722c7943dcce356335054

File: ui/pages/liquidity.tsx [hunk 3]
```diff
@@ -337,8 +337,23 @@ export default function Liquidity() {
...
                       action={withdrawAndClaimRewards}
+                      preAction={() =>
+                        userDeposit &&
+                        setWithdrawalValue(
+                          transformNumber(
+                            userDeposit,
+                            NumberType.string
+                          ) as string
+                        )
+                      }
```
Comment (johnmark13) [inline ui/pages/liquidity.tsx:348]: I think as this is now the same code as `max` I'd have pulled out a function, but I also think this is a neat solution so I won't lose sleep!

**Review by johnmark13 — APPROVED** [review]

**Comment by aahna-ashina** [comment]
> all attempts to deposit on goerli fail for some reason

@Caranell Depositing pool tokens did not work for you on Goerli?  What errors are you seeing?

File: contracts/deployments/goerli.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
   "nationDropContracts": ["0xCBbd5a8871E6Cf995305e58168cEaFB0be512c67"],
   "balancerVault": "0xba12222222228d8ba445958a75a0704d566bf2c8",
   "balancerPoolId": "0x6f57329d43f3de9ff39d4424576db920b55060b30002000000000000000000d7",
-  "balancerLPToken": "0x6f57329d43f3de9ff39d4424576db920b55060b3",
+  "balancerLPToken": "0x6f57329D43f3De9fF39d4424576dB920b55060B3",
```
Comment (aahna-ashina) [inline contracts/deployments/goerli.json:7]:
```suggestion
  "balancerLPToken": "0x6f57329d43f3de9ff39d4424576db920b55060b3",
```

**Comment by Caranell** [comment]
@aahna-ashina I don't really know what's the problem cause the contract is not verified, there're multiple failed deposit transactions in etherscan -- https://goerli.etherscan.io/address/0x165a7052695adb159660d59f61db0c03253ac526

**Comment by aahna-ashina** [comment]
> I don't really know what's the problem cause the contract is not verified, there're multiple failed deposit transactions in etherscan -- https://goerli.etherscan.io/address/0x165a7052695adb159660d59f61db0c03253ac526

@Caranell See https://github.com/nation3/app/pull/80, which was merged earlier today.

We switched to a Balancer Vault contract which is verified.  Here is the new URL:  https://goerli.balancer.fi/#/pool/0x6f57329d43f3de9ff39d4424576db920b55060b30002000000000000000000d7

**Comment by Caranell** [comment]
I guess we need to wait for the redeployed contract https://github.com/nation3/app/pull/80#pullrequestreview-1075323139 

As it doesn't really connected to the original task, maybe it's time to merge this one?

**Pull request merged**